app/assets/stylesheets/new_entry_form.css

```diff
@@ -0,0 +1,33 @@
+/*new entry page css*/
+
+/*COLORS*/
+/* #FAED26 - yellow */
+/* #46344E - indigo */
+/* #5A5560 - blue gray */
+/* #9D8D8F - paper */
+/* #9B786F - latte */
+
+.mainPanel {
+	margin: 0px;
+	padding: 30px 0px 0px 30px;
+}
+p {
```

Comment:
style on <p> will affect all elements - generally this is a little scary.

Maybe in the future we should consider using SASS